ddtrace/contrib/dbapi/__init__.py

```diff
@@ -34,7 +36,13 @@ def __init__(self, cursor, pin, cfg):
         name = pin.app or 'sql'
         self._self_datadog_name = '{}.query'.format(name)
         self._self_last_execute_operation = None
-        self._self_config = cfg or config.dbapi2
+        if not cfg:
+            debtcollector.deprecate(
+                "ddtrace.config.dbapi2 is now deprecated, use an integration config for TracedCursor",
+                removal_version="0.48.0",
```

Comment:
0.48 is the next release which means we're both deprecating and removing it in the same version?

Reply:
Probably also makes sense to do the deprecation in separate PR as it's not really related to the optimization

Reply:
👍🏽 #2111